configure.ac

```diff
@@ -88,7 +93,11 @@ PKG_PROG_PKG_CONFIG
 #
 # Do we have the brew command from Homebrew?
 #
-AC_PATH_PROG([BREW], [brew])
+case "$host_os" in
```

Comment:
Homebrew also supports Linux...

...but I'm not sure whether anybody would use it rather than the native package manager that almost all if not all Linux distributions have, so there's probably no reason to test for it on Linux.

Reply:
But there's probably no problem with testing for ti.

Reply:
This eliminates some tests that are never true, as far as the developers of these optional dependencies see it.